packages/Popover/CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## [Unreleased]
 
+### Added
+
+- Added `aria-haspopup` to trigger component
```

Comment:
You can skip this step no, since the content fo the changeset gets added to the CHANGELOG.